experiment/simulation/index.html

```diff
@@ -1,16 +1,133 @@
+								Task 2
+							</a>
+						</li>
+						<li id="Task3" onclick="changeTabs(event)">
+							<a>
+								Task 3
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+
+		<div class="columns is-centered">
+			<div class="column">
+				<div class="v-instruction-container">
+					<div class="v-instruction-title" id="TaskTitle">Half Adder</div>
+					<div class="v-instruction-text" id="TaskDescription">
+						Implement a 1-bit half adder
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="columns is-centered">
+		<div class="column is-2-desktop is-4-tablet is-12-mobile">
+			<div class="v-datalist-container components-list">
+				<div class="v-datalist-title">Components</div>
+				<div class="v-datalist-row">
+					<div class="columns is-centered" id="toolbar">
+						<div class="column is-6-desktop is-6-tablet is-6-mobile">
+							<div class="component-button AND" onclick="Add(event)">AND</div>
+							<div class="component-button OR" onclick="Add(event)">OR</div>
+							<div class="component-button NOT" onclick="Add(event)">NOT</div>
+							<div class="component-button NAND" onclick="Add(event)">NAND</div>
+						</div>
+						<div class="column is-6-desktop is-6-tablet is-6-mobile">
+							<div class="component-button NOR" onclick="Add(event)">NOR</div>
+							<div class="component-button XOR" onclick="Add(event)">XOR</div>
+							<div class="component-button XNOR" onclick="Add(event)">XNOR</div>
+						</div>
+					</div>
+				</div>
+
+			</div>
+		</div>
+		<div class="column is-7-desktop is-12-tablet is-12-mobile">
+			<div id="circuit-board">
+				<div id="working-area">
+				</div>
+			</div>
+		</div>
+		<div class="column is-3-desktop is-4-tablet is-12-mobile">
+			<div class="v-datalist-container components-list">
+				<div class="v-datalist-title">Observations</div>
+				<div class="v-datalist-row">
+					<div class="v-table-wrap">
+						<table class="table is-bordered is-fullwidth">
+							<thead id="table-head">
+								<tr>
+									<th colspan="2">Inputs</th>
+									<th colspan="2">Expected Values</th>
+									<th colspan="2">Observed Values</th>
+								</tr>
+								<tr>
+									<th>A</th>
+									<th>B</th>
+									<th>Sum</th>
+									<th>Carry</th>
+									<th>Sum</th>
+									<th>Carry</th>
+								</tr>
+							</thead>
+							<tbody id="table-body">
+							</tbody>
+						</table>
+					</div>
+					<div id="result"></div>
+				</div>
+				<div class="v-datalist-row">
+					<button class="v-button" onclick="simulate()">Simulate</button>
+					<button class="v-button" onclick="submitCircuit()">Submit</button>
+				</div>
+
+			</div>
+		</div>
+	</div>
+	<div class="menu">
+		<ul class="menu-options">
+			<li class="menu-option">Delete</li>
+			<li class="menu-option">Cancel</li>
+		</ul>
+	</div>
+
+</body>
+<script src="js/jsplumb.bundle.js"></script>
+<!-- <script src="js/layout.js"></script> -->
+<script type="module" src="js/main.js"></script>

```

Comment:
Should the imports of jsplumb and main be inside the \<body\> tag?

Reply:
yes, I have updated the file